### PR TITLE
DOCS-1948-CE-3.15.5

### PR DESCRIPTION
**Pull request by bmckercher123**
Patch release.

Product Version(s):
CE 3.15.5

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.15/releases.json [hunk 1]
```diff
@@ -1,4 +1,257 @@
 [
+  {
+    "title": "v3.15.5",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "version": "v1.28.14",
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.15/releases.json]: v1.28.15

File: calico-enterprise_versioned_docs/version-3.15/releases.json [hunk 2]
```diff
@@ -1,4 +1,257 @@
...
+        "image": "tigera/honeypod-exp-service",
+        "version": "v3.15.5"
+      },
+      "honeypod-controller": {
+        "image": "tigera/honeypod-controller",
+        "version": "v3.15.5"
+      },
+      "key-cert-provisioner": {
+        "image": "tigera/key-cert-provisioner",
+        "version": "v1.1.11",
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.15/releases.json]: v1.1.14

File: calico-enterprise_versioned_docs/version-3.15/releases.json [hunk 3]
```diff
@@ -1,4 +1,257 @@
...
+      "fluentd-windows": {
+        "image": "tigera/fluentd-windows",
+        "version": "v3.15.5"
+      },
+      "es-proxy": {
+        "image": "tigera/es-proxy",
+        "version": "v3.15.5"
+      },
+      "eck-kibana": {
+        "version": "7.17.11"
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.15/releases.json]: 7.17.14

File: calico-enterprise_versioned_docs/version-3.15/releases.json [hunk 4]
```diff
@@ -1,4 +1,257 @@
...
+      },
+      "kibana": {
+        "image": "tigera/kibana",
+        "version": "v3.15.5"
+      },
+      "eck-elasticsearch": {
+        "version": "7.17.11"
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.15/releases.json]: 7.17.14

File: calico-enterprise_versioned_docs/version-3.15/releases.json [hunk 5]
```diff
@@ -1,4 +1,257 @@
...
+      "packetcapture": {
+        "image": "tigera/packetcapture",
+        "version": "v3.15.5"
+      },
+      "prometheus": {
+        "image": "tigera/prometheus",
+        "version": "v3.15.5"
+      },
+      "coreos-prometheus": {
+        "version": "v2.32.2"
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.15/releases.json]: v2.47.0

File: calico-enterprise_versioned_docs/version-3.15/releases.json [hunk 6]
```diff
@@ -1,4 +1,257 @@
...
+      },
+      "eck-elasticsearch-operator": {
+        "version": "2.6.1"
+      },
+      "elasticsearch-operator": {
+        "image": "tigera/eck-operator",
+        "version": "v3.15.5"
+      },
+      "coreos-alertmanager": {
+        "version": "v0.23.0"
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.15/releases.json]: v0.25.1

**Review by danudey — APPROVED** [review]
Just these version changes and then everything seems good!

File: calico-enterprise_versioned_docs/version-3.15/releases.json (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.15/releases.json (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.15/releases.json (hunk 3 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.15/releases.json (hunk 4 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.15/releases.json (hunk 5 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.15/releases.json (hunk 6 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/releases.json]: Fixed.

**Pull request merged**